### ISSUE TITLE
MC Scale (Inflate Cube) is hard capped at -7.000/7.000

### ISSUE THREAD

**Issue by FelisMoon** [tom5454/CustomPlayerModels#610]
**_TLDR: Get rid off the cap. Users already get around it._**

MC Scale at very high values (both positive and negative) can be used for several creative rendering tricks. The current cap it has complicates and makes it harder to perform them, as this can be bypassed with Blockbench exports.

Here is an image of a copy of the head at _-7.000_ (MinCap) to compare sizes with the actual head. Negative cubes have special rendering properties and effects, and the maximum size it can get on its own is really small and isn't even able to match its dimensions.
![image](https://github.com/tom5454/CustomPlayerModels/assets/81988520/e4387982-e539-4e93-bd8e-dca5dc32089f)

Here is an image of a copy of the head at _7.000_ (MaxCap) to compare sizes yet again. Although more loose, the cap still forbids users from making cubes as big as they wished for, which may be intentional. except, it doesn't. (more on this below)
![image](https://github.com/tom5454/CustomPlayerModels/assets/81988520/ac373aad-1b45-4172-a82f-5e2c0b07b823)

 I can maybe understand this cap was added because its unorthodox to use such big sizes by minecraft standards and most importantly, so users don't make obnoxiously enormous models that have huge invasive cubes to troll or cause visual problems to other players. But by stacking a good amount of features, this can still be done.
 
 THIS: Is a copy of the head that has MAX Size, MCscale AND AdvancedScaling. If the server has pehkui or public access to attributes in modern versions, the display of this enormous cube can be even bigger. And this has been done only using CPM, it could be pushed even further with Blockbench exports.
![image](https://github.com/tom5454/CustomPlayerModels/assets/81988520/d1b4a30a-74f4-4682-80b0-b8e598f8f4e8)

As mentioned before, with the help of Blockbench, this CAP can be bypassed with its uncapped inflate.
![image](https://github.com/tom5454/CustomPlayerModels/assets/81988520/111c49bd-8872-473d-8375-ecbfa8b591f0)
![image](https://github.com/tom5454/CustomPlayerModels/assets/81988520/d0840db6-2992-4292-af4e-fdf18404aba4)

And used for tricks like these, which should be able to be done WITHOUT the need of Blockbech exports.
![image](https://github.com/tom5454/CustomPlayerModels/assets/81988520/cf9b7ec0-5e88-4d6a-bf87-21d6af052248)

I feel like this cap has to go in order to allow the same level of creativity other users are able to get with use of blockbench or file editing, and other different measure has to be used to fight players who want to use the mod to cause issues. For example, adding further safety tools.
